e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/security/accesscontrol/deleteput/ResourceDto.java

```diff
@@ -0,0 +1,17 @@
+package com.foo.rest.examples.spring.security.accesscontrol.deleteput;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class ResourceDto {
+
+    @ApiModelProperty
+    public String stringValue;
+
+    @ApiModelProperty
+    public int integerValue;
```

Comment:
never use `int` and `boolean` in DTOs, as they assume not `null`.
rather use `Integer` and `Boolean`

Reply:
Hi Andrea, thanks for your comment. I changed types to Integer and Boolean as suggested.